### PR TITLE
Update hermes docs to use bundled hermesc script

### PR DESCRIPTION
**Pull request by SMJ93**
Since React Native 0.69 hermes has been [bundled with React Native](https://reactnative.dev/blog/2022/06/21/version-069#bundled-hermes).

If users use the old `hermesc` version from  `node_modules/hermes-engine` the version won't match.

This PR recommends users to change their hermes script to use the the `hermesc` version from `react-native` if they are using React Native 0.69 or later:

```
// OS-BIN is osx-bin, win64-bin, or linux64-bin, depending on which operating system you are using
node_modules/react-native/sdks/hermesc/{OS-BIN}/hermesc
```

### Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

### CODE REVIEW

**Comment by imatwawana** [comment]
It looks like some of what you've added is already covered in a note just before step 3, but the folks on @getsentry/team-mobile will need to check this for technical accuracy. After that, I'll review for wording and structure.

**Comment by SMJ93** [comment]
> It looks like some of what you've added is already covered in a note just before step 3, but the folks on @getsentry/team-mobile will need to check this for technical accuracy. After that, I'll review for wording and structure.

Hi @imatwawana,

Yeah, that is quite hidden away - it is also better to explain why it has changed (see the link to react native blog).

**Comment by imatwawana** [comment]
> > It looks like some of what you've added is already covered in a note just before step 3, but the folks on @getsentry/team-mobile will need to check this for technical accuracy. After that, I'll review for wording and structure.
> 
> Hi @imatwawana,
> 
> Yeah, that is quite hidden away - it is also better to explain why it has changed (see the link to react native blog).
> 
> Maybe a better alternative would be to flip the default to React Native 0.69 and keep the note for 0.68 and below?

I'll definitely defer to the @getsentry/team-mobile team on the best way to approach this.

**Comment by marandaneto** [comment]
I'd say we keep this open till more people adopt the 0.69.x version, right now the majority of our users are not there yet.
The `note` below mentions 0.69.x support, what we can do is to move the `note` above the command, what do you think @imatwawana ?

**Comment by imatwawana** [comment]
Sure, let's move the note up.

**Comment by imatwawana** [comment]
Feel free to just update this PR

**Comment by imatwawana** [comment]
Oh sorry, I just realized that bumps into the recommendation that @marandaneto had made. Let's leave this one open and I'll open a new PR to move the note. I'll circle back and make some suggestions on this PR to get it ready for when we can use it down the road.

File: src/platforms/react-native/manual-setup/hermes.mdx
Comment (imatwawana) [inline src/platforms/react-native/manual-setup/hermes.mdx:67]:
```suggestion
If you're using Hermes with **React Native 0.68 or below**:
```

File: src/platforms/react-native/manual-setup/hermes.mdx
Comment (imatwawana) [inline src/platforms/react-native/manual-setup/hermes.mdx:75]:
```suggestion
Starting with **React Native 0.69** `hermes` has been [shipped with React Native](https://reactnative.dev/blog/2022/06/21/version-069#bundled-hermes), so you should use `hermesc` from `react-native`:
```

**Review by imatwawana — COMMENTED** [review]
I've made some wording suggestions. You'll also need to delete the note starting at line 82 so that the snippet you've added isn't redundant. I'll review again when you've committed changes.

File: src/platforms/react-native/manual-setup/hermes.mdx
Comment (imatwawana) [inline src/platforms/react-native/manual-setup/hermes.mdx:75]:
```suggestion
Starting with **React Native 0.69**, `hermes` has been [shipped with React Native](https://reactnative.dev/blog/2022/06/21/version-069#bundled-hermes), so you should use `hermesc` from `react-native`:
```

**Review by imatwawana — APPROVED** [review]
I missed a comma, but otherwise this looks good. We'll just leave this open until such time as @marandaneto thinks we should merge it.

**Comment by SMJ93** [comment]
Hey @imatwawana, can this be merged?

**Comment by imatwawana** [comment]
> Hey @imatwawana, can this be merged?

Per @marandaneto's comment before, we're gonna hold off on merging until more folks have adopted the newer version:
https://github.com/getsentry/sentry-docs/pull/5455#issuecomment-1231336075
@marandaneto, please merge this whenever it makes sense.

**Comment by SMJ93** [comment]
@imatwawana any update with this?

**Comment by SMJ93** [comment]
@imatwawana @marandaneto any update with this?

**Comment by marandaneto** [comment]
> @imatwawana @marandaneto any update with this?

Sentry RN SDK v5 is still alpha (currently working on it), also the adoption of RN >= 0.70 is very low, any reason why you would like to push this? the docs already state which commands to run on the older and newer version.

**Comment by SMJ93** [comment]
@marandaneto the adoption rate of 0.69+ is pretty high which includes the bundled version of hermes.

Chasing as I created this PR nearly 3 months ago.

Feel free to close if you are updating the cods as part of v5.

**Comment by krystofwoldrich** [comment]
@marandaneto Maybe we can make this change independent of RN SDK v5 as v4 users can use v4 with 0.69+ if they make changes described in the troubleshooting page.

**Comment by SMJ93** [comment]
@krystofwoldrich @marandaneto This is probably out of date now - feel free to close

**Comment by marandaneto** [comment]
@krystofwoldrich adjust the text if necessary and merge it since RN 0.7x is already stable.

**Review by krystofwoldrich — APPROVED** [review]
@marandaneto @SMJ93 It still applies. It looks good.

**Pull request merged**
